### DISCUSSION TITLE
Finder failed with msg:  vim/shared.lua:0: prefix: expected string, got nil

### DISCUSSION THREAD

**Discussion by reet-** [cljoly/telescope-git-repo.nvim#72] [Q&A] [answered]
I updated NVIM to v0.10.0-dev-2698+g00e71d3da and telescope repo to `ca8a541eeb9c` yesterday, and I'm now getting the following error with `:Telescope repo list`:

```
[telescope] [WARN  17:00:33] .local/share/nvim/lazy/telescope.nvim/lua/telescope/pickers.lua:662: Finder failed with msg:  vim/shared.lua:0: prefix: expected string, got nil 
```

```lua
local ok, msg = pcall(function()
  self.finder(prompt, process_result, process_complete)
end)

if not ok then
  log.warn("Finder failed with msg: ", msg)
end
```

The effect is that I do not see any repositories. The identical config worked before the update, any help is appreciated. Thanks!

**Comment by cljoly** [comment, marked as answer]
This might be fixed now that #75 is merged. Could you please check again @reet-?

**Comment by reet-** [comment]
Works again, great! Thanks @cljoly

**Reply by cljoly** [reply]
No worries, thanks for checking.
